ddl/ddl_worker.go

```diff
@@ -518,8 +518,10 @@ func newMetaWithQueueTp(txn kv.Transaction, tp workerType) *meta.Meta {
 	return meta.NewMeta(txn)
 }
 
-func (w *worker) setDDLLabelForTopSQL(job *model.Job) {
+func (w *jobContext) setDDLLabelForTopSQL(job *model.Job) {
 	if !topsqlstate.TopSQLEnabled() || job == nil {
+		w.cacheDigest = nil
+		w.ddlJobCtx = context.Background()
```

Comment:
Why reset cacheDigest?

Reply:
When topsql is disabled, should reset the `ddlJobCtx`, since some internal SQL use will use this context to execute SQL. Reset the `cacheDigest` is just in case.

Reply:
Will `cacheDigest` be modified?

Reply:
No, actually `cacheDigest` won't be modified/read in other place, so `cacheDigest` no need to be reset here, but I reset here just in case in the future.

Reply:
I am concerned about the cost of generating a new `cacheDigest`.

Reply:
No, normally it won't generating a new `cacheDigest`.